pkgs/by-name/un/unyaffs/package.nix

```diff
@@ -0,0 +1,45 @@
+{
+  fetchFromGitHub,
+  lib,
+  stdenv,
+  versionCheckHook,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "unyaffs";
+  version = "0.9";
+
+  src = fetchFromGitHub {
+    owner = "whataday";
+    repo = "unyaffs";
+    rev = "${finalAttrs.version}";
+    hash = "sha256-FcuPaxq74gVJ6UlOhFPSMtuwUAJVV/sAxAQojhmVXCs=";
+  };
+
+  preBuild = ''
+    sed -i '54i #if defined(__linux__)' unyaffs.c
+    sed -i '55i #include <sys/sysmacros.h>' unyaffs.c
+    sed -i '56i #endif' unyaffs.c
+  '';
+
+  installPhase = ''
+    runHook preInstall
+
+    install -Dm755 unyaffs -t $out/bin
+
+    runHook postInstall
+  '';
+
+  nativeInstallCheckInputs = [ versionCheckHook ];
+  doInstallCheck = true;
+  versionCheckProgramArg = "-V";
+
+  meta = {
+    description = "Tool to extract files from a YAFFS2 file system image";
+    homepage = "https://github.com/whataday/unyaffs";
+    license = lib.licenses.gpl2Only;
+    platforms = lib.platforms.unix;
+    maintainers = with lib.maintainers; [ KSJ2000 ];
```

Comment:
Should be good now.

Reply:
I'm not sure why the ofborg-eval is failing.
I thought it was because of the extra "" I added for githubID that others didn't, but it was not the case. Ofborg failure was the same afterwards, so I don't know what I'm supposed to do.